Source/Evolve/WarpXEvolve.cpp

```diff
@@ -569,7 +569,7 @@ void WarpX::SyncCurrentAndRho ()
             {
                 // TODO This works only without mesh refinement
                 const int lev = 0;
-                if (use_filter) { ApplyFilterJ(current_fp_vay, lev); }
+                if (use_filter) { ApplyFilterMF(current_fp_vay, lev); }
             }
```

Comment:
Should ApplyFilterJ() be different than applying a filter for E or B when using curvilinear geometries? 

I believe the filter should be applied to the current before dividing by the volume element to convert to current density. Perhaps this is being done somewhere else?